### PR TITLE
add voided.gg

### PR DESCRIPTION
**Pull request by J4C0B3Y**
## Type of change

- [x] Add new directory for a server
- [ ] Update directory of a server
- [ ] Remove directory of a server
- [ ] Documentation Update

## Checklist

- [x] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [x] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [x] I have created the manifest.json with the required values

### CODE REVIEW

File: minecraft_servers/voided/background.png
Comment (maaaathis) [inline minecraft_servers/voided/background.png:1]: Compress this file

File: minecraft_servers/voided/background@2x.png
Comment (maaaathis) [inline minecraft_servers/voided/background@2x.png:1]: Compress this file

File: minecraft_servers/voided/manifest.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
+{
+  "server_name": "voided",
+  "nice_name": "Voided Network",
+  "direct_ip": "voided.gg",
+  "server_wildcards": ["%.voided.gg"],
```
Comment (maaaathis) [inline minecraft_servers/voided/manifest.json:5]: Doesn't look like a real wildcard
image

File: minecraft_servers/voided/manifest.json [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  "supported_languages": ["en"],
+  "social": {
+    "web": "https://voided.gg",
+    "web_shop": "https://store.voided.gg",
```
Comment (maaaathis) [inline minecraft_servers/voided/manifest.json:9]: Returns a `DNS_PROBE_FINISHED_NXDOMAIN` error

**Review by maaaathis — CHANGES_REQUESTED** [review]

File: minecraft_servers/voided/manifest.json (hunk 2 above)
Comment (DoJapHD) [reply minecraft_servers/voided/manifest.json:9]: Still not available

**Review by DoJapHD — CHANGES_REQUESTED** [review]

**Comment by J4C0B3Y** [comment]
I've compressed the images and fixed the store links, should be all good now.

**Review by DoJapHD — APPROVED** [review]

**Pull request merged**